tests/basic/upgrade_cluster_test.go

```diff
@@ -90,6 +91,7 @@ var _ = Describe("{UpgradeCluster}", func() {
 
 				err = Inst().S.UpgradeScheduler(version)
 				if err != nil {
+					err = Inst().S.RefreshNodeRegistry()
```

Comment:
Do you need to check this error? If not we can just do `Inst().S.RefreshNodeRegistry()`

Reply:
Actually Anthos scheduler is invoking "RefreshNodeRegistry()"  after the upgrade completed. This method only update new nodes IPs and names but doesn't update the flag `IsStorageDriverInstalled` because of this when `GetStorageNodes()` method invoked after upgrade, it return empty list (As part of `PrintCommandOutput()` method call `node.GetStorageNodes()[0]`)

Reply:
Sorry I need to use this method `Inst().V.RefreshDriverEndpoints()`